src/core.js

```diff
@@ -36,7 +36,7 @@ QUnit.isLocal = !( defined.document && window.location.protocol !== "file:" );
 // Expose the current QUnit version
 QUnit.version = "@VERSION";
 
-function createModule( name, testEnvironment ) {
+function createModule( name, testEnvironment, skip ) {
```

Comment:
The third argument here should likely be an options hash instead of a single value. That way we don't have to add an additional argument when we do `todo` (and who knows if we'll add something else in the future!)